### DISCUSSION TITLE
Shortcut in Windows menu

### DISCUSSION THREAD

**Discussion by SternMSF** [xournalpp/xournalpp#2663] [Q&A] [answered]
I installed Xournal++ yesterday, and find there is no shortcut created in the Windows menu, so I have to launch with a command line instruction.
Is there a way to fix this? Did I skip something in the install?
platform Win 10
xpp version 1.0.20

**Comment by Technius** [comment]
Installer should automatically place a shortcut in the start menu under the `Xournal++` folder. If not, you might be running into a bug.

**Reply by SternMSF** [reply]
I will try to reinstall, and report back.

**Reply by Technius** [reply, marked as answer]
The current installer has a bug where it doesn't delete the old Xournal++ directory, so you'll want to run the `Uninstall.exe` first and then run the installer.
